pkg/progress/action.go

```diff
@@ -144,6 +144,26 @@ func completedOrFailed(aIDX int, actionSet *crv1alpha1.ActionSet, phaseName stri
 	return false
 }
 
+func isPhaseProgressDiffer(a, b crv1alpha1.PhaseProgress) bool {
```

Comment:
I am not able to figure out what exactly this function is, by reading the name. is it trying to figure out if phase progress is different?

Reply:
`a` and `b` both are phase progress, the function returns true if they are different, and false if they are similar.

Reply:
I can't think of a better name, do you have any ideas?

Reply:
maybe having different instead of differ would be a good idea? Or if we want to shorten we can just use `Diff` instead of differ.